### PR TITLE
first cosmic velocity

### PR DESCRIPTION
**Pull request by aokiyoshi**
rule and tests added

### CODE REVIEW

File: test/gravity/first_cosmic_velocity_test.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    result = first_cosmic_velocity.calculate_velocity(
+        test_args.m, test_args.r, test_args.h
+    )
+    assert SI.get_dimension_system().equivalent_dims(result.dimension, units.velocity)
+    result_velocity = convert_to(result, units.meter / units.second).evalf(4)
+    assert result_velocity == approx(7910, 0.001)
+
+
+def test_bad_mass(test_args):
+    bm = Quantity(5000000 * units.coulomb)
```
Comment (blackyblack) [inline test/gravity/first_cosmic_velocity_test.py:34]: Make it 1.

File: symplyphysics/laws/gravity/first_cosmic_velocity.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+
+# Law: V = √(G * M / (R + h))
+# Where:
+# V - initial velocity
+# M - planet_mass of planet
+# G - gravitational constant
+# h - height above the planet surface
```
Comment (blackyblack) [inline symplyphysics/laws/gravity/first_cosmic_velocity.py:12]: What is R?

File: symplyphysics/laws/gravity/first_cosmic_velocity.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+velocity = Symbol("initial_velocity", units.velocity)
+planet_mass = Symbol("planet_mass", units.mass)
+radius = Symbol("radius", units.length)
+height = Symbol("height", units.length)
+
+law = Eq(velocity, sqrt(gravitational_constant * planet_mass / (radius + height)))
```
Comment (blackyblack) [inline symplyphysics/laws/gravity/first_cosmic_velocity.py:19]: I think it should be derived quite easily. Do you want to try and prove it?

File: symplyphysics/laws/gravity/first_cosmic_velocity.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+from sympy import Eq, solve, sqrt
```
Comment (blackyblack) [inline symplyphysics/laws/gravity/first_cosmic_velocity.py:1]: Avoid marketing names for laws. Rename for people who don't know what is first cosmic velocity.

Comment (aokiyoshi) [reply symplyphysics/laws/gravity/first_cosmic_velocity.py:1]: I don't know how to call it, because there are 4 cosmic velocities and all named same.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**